hooks/pre-commit.hook

```diff
@@ -0,0 +1 @@
+git diff -U0 HEAD~2 HEAD | data/check-style --diff
```

Comment:
I'm confused as to why there's both hooks/pre-commit and hooks/pre-commit.hook - it seems the latter is redundant?